### PR TITLE
Py3

### PR DESCRIPTION
**Pull request by jburel**
* Require py3
* update travis

### CODE REVIEW

**Review by mtbc — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -15,5 +15,6 @@ deploy:
   provider: pypi
   user: $PYPI_USER
   password: $PYPI_PASSWORD
+  distributions: sdist bdist_wheel
```
Comment (sbesson) [inline .travis.yml:18]: Not sure this is strictly necessary given there is no source code in this package but it probably does not hurt

**Review by sbesson — COMMENTED** [review]
No objection to bumping the requirement to Python 3 here to match the rest of the Python packages. Should this be tagged and released as `0.5.0`? Main caveat is that this might affect all places where the package has been capped as described in the readme

**Comment by manics** [comment]
It's fine to merge, but I don't see any reason to make a release until we bump the dependencies (e.g. We'll need to bump Ansible to get full python 3 support on remote hosts)

**Review by sbesson — APPROVED** [review]
With https://github.com/ome/ansible-roles/pull/14, it should be now safe to get this merged as the latest version of the `ome-ansible-molecule` package is installed in a Python 3 environment

**Pull request merged**